addons/website_sale_slides/views/website_slides_templates.xml

```diff
@@ -105,10 +105,8 @@
 
 <!-- TOOLS -->
 <template name="Buy Course Link" id="course_buy_course_link">
-    <a t-att-href="'/shop/cart/update?product_id=%s' % slide.channel_id.product_id.id">
-        Buy Course
-    </a>
-     to download resources
+    <a class="post_link" t-att-href="'/shop/cart/update?product_id=%s' % slide.channel_id.product_id.id">
+        Buy Course</a> to download resources
```

Comment:
"post_link" is fine.

However, does re-formatting the line break the translations?
Not sure, maybe best to avoid doing it or double-check by re-generating the PO file.